dbt_project.yml

```diff
@@ -26,3 +26,4 @@ on-run-end:
   - "{{ elementary.upload_dbt_artifacts() if not elementary.get_config_var('disable_dbt_artifacts_autoupload') }}"
```

Comment:
General comment - what do you think about just adding the elementary package version to the new dbt invocations table instead of using a new table? This way you will have history of the upgrades side by side with the dbt versions and you won't need to add additional step to the on-run-end which becomes bigger and bigger.

Reply:
Sounds pretty good. I'll do that.
I'll have to update Dor as well.